src/io.rs

```diff
@@ -41,13 +41,13 @@ where
 {
     /// Create a new buffered `MultiLineWriter` instance that suffixes
     /// each write with a newline character ('\n').
-    pub(crate) fn new(cap: usize, inner: T) -> MultiLineWriter<T> {
-        Self::with_ending(cap, inner, "\n")
+    pub(crate) fn new(inner: T, cap: usize) -> MultiLineWriter<T> {
```

Comment:
Still not a huge fan of changing argument ordering without a massively important reason (kind of like how you only add new arguments to the end to avoid possible issues with FFI calls.)

Still, your library, and not a lot of things *should* be calling this yet.

Reply:
> Still not a huge fan of changing argument ordering without a massively important reason

It's internal to the crate so nothing _could_ be calling it.

Thanks for the review!

Reply:
Well, that certainly is a good reason. (sorry, too many code reviews lately and bad brain).